politico_api/tests/test_office_get.py

```diff
@@ -1,10 +1,10 @@
 # allow for import from the politico_app folder 
 import sys
-sys.path.insert(0,'../..')
-
 import json
-from politico_api.config import app 
 import unittest
+from politico_api.v1 import create_app
+app = create_app()
+
```

Comment:
Instead of creating the app for every test file, you can have a base config file that will setup the app then you import it into every file

Reply:
If I understand you, you're saying base config to be only by files that are testing?

Reply:
The base config should have the initialisation for the testing app then you inherit it in the other files